uploader/pantheon.py

```diff
@@ -299,22 +300,26 @@ def process_workspace(path):
     logger.debug('url: %s', url)
     workspace = {}
     workspace['jcr:primaryType'] = 'pant:workspace'
+    workspace['jcr:lastModified'] = datetime.now().strftime("%Y-%m-%dT%H:%m:%S") #SimpleDateFormat:yyyy-MM-dd'T'HH:mm:ss
```

Comment:
Do we need update the jcr:lastModified timestamp. My understanding is that sling populates for us by default

Reply:
I see there might be a reason we need update the jcr:lastModified timestamp where same repo is re-uploaded with modifications. the timestamp you populated here is based off the timezone on the client size. On the serverside the time is UTC based. If you have to generate the modified timestamp on the client side, you should ensure it's utc time.

Reply:
Added changes to convert localtime to UTC in latest commit